### ISSUE TITLE
[Bug]: Narrator Support Issue - Electron App does not emit property/structure changed event

### ISSUE THREAD

**Issue by mliao95** [electron/electron#28479]
### Preflight Checklist

I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Electron Version

12.0.0

### What operating system are you using?

Windows

### Operating System Version

Windows 10

### What arch are you using?

x64

### Expected Behavior

In an Electron app, Narrator should be able to announce to the user when a checkbox is toggled, an element is expanded/collapsed or when a button is pressed.

### Actual Behavior

Narrator is able to announce elements when they gain focus, but fails to announce any state changes such as the ones listed in the expected behavior.

I have noticed some Narrator support issues for Electron apps pertaining to state changes for an element.

Repro steps:
- create a sample electron app with checkboxes, expandable/collapsible elements and toggle-able buttons.
- Open Narrator
- Try to toggle any of the elements

Expected behavior:
Narrator should announce "checkbox checked/unchecked", "element expanded/collapsed", etc.

Actual behavior:
Narrator stays silent

Talking with the Narrator team, it looks like there is no property/structure changed event firing from the Application.

**Comment by nornagon** [comment]
Thanks for reporting this and helping to make Electron better!

Would it be possible for you to make a standalone testcase with only the code necessary to reproduce the issue? For example, [Electron Fiddle](https://www.electronjs.org/fiddle) is a great tool for making small test cases and makes it easy to publish your test case to a [gist](https://gist.github.com) that Electron maintainers can use.

Stand-alone test cases make fixing issues go more smoothly: it ensure everyone's looking at the same issue, it removes all unnecessary variables from the equation, and it can also provide the basis for automated regression tests.

I'm adding the `blocked/need-repro` label for this reason. After you make a test case, please link to it in a followup comment.

Thanks in advance! Your help is appreciated.

**Comment by mliao95** [comment]
Didn't even know Electron Fiddle existed.  Very cool!  

I made a [simple test case](https://gist.github.com/mliao95/acb071de0837118c83586a7adb0d21f3) with just a checkbox as I assume the issue with the checkbox is the same as the other toggle-able elements. Let me know if this is the format you wanted it in or if you need anything else from me.

I noticed that there was an [old issue](https://github.com/electron/electron/issues/11867) targeting Narrator compatibility that pointed out the main issue is Narrator's reliance on UIAutomation APIs to read the screen.  I'm guessing the root cause of this issue stems from that as well.

**Comment by blueset** [comment]
Can reproduce this issue in Electron 20.3.3.

**Comment by JLSeto** [comment]
bump

**Comment by msftedad** [comment]
We have verified this issue in latest environment and issue able to reproduce. Can you please activate this issue.

**Comment by msftedad** [comment]
Hi Team,

Any update on this ticket?

**Comment by msftedad** [comment]
Verified in latest VS Code and issue is reproducible. Please refer attachment(Repro_Narrator_VSCode.webm)
Verified in:
Win11:
OS version: 22H2
OS build: 25370.1000
VS code version(Insiders): 1.78.2

**Comment by msftedad** [comment]
Hi @mliao95, Issue is still repro in latest Environment. As per below comment, we are able to reproduce this issue on [Supported version of Electron](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#timeline)(27.3.2). Could you please help us in reactivating the External issue. Providing attachment for reference(Repro_toggle element on off.webm)
Verified in:
Win11:
OS version: 24H2
OS build: 26063.1
VS code version(Insiders): 1.87.2
Electron: 27.3.2
Could you please reopen this issue.

**Comment by mliao95** [comment]
Hi @msftedad, I don't appear to have the permissions to re-open this issue. You would probably need a project maintainer to re-open it or create a new issue and reference this one.
